src/QMCWaveFunctions/Fermion/DelayedUpdate.h

```diff
@@ -19,179 +19,271 @@
+    const int norb     = Ainv.rows();
+    const int lda_Binv = Binv.cols();
+    // save Ainv[rowchanged] to invRow
+    std::copy_n(Ainv[rowchanged], norb, invRow.data());
+    // multiply V (NxK) Binv(KxK) U(KxN) invRow right to the left
+    BLAS::gemv('T', norb, delay_count, cone, U.data(), norb, invRow.data(), 1, czero, p.data(), 1);
+    BLAS::gemv('N', delay_count, delay_count, cone, Binv.data(), lda_Binv, p.data(), 1, czero, Binv[delay_count], 1);
+    BLAS::gemv('N', norb, delay_count, -cone, V.data(), norb, Binv[delay_count], 1, cone, invRow.data(), 1);
+  }
 
-      /** accept a move with the update delayed
+  /** accept a move with the update delayed
        * @param Ainv inverse matrix
        * @param rowchanged the row id corresponding to the proposed electron
        * @param psiV new orbital values
        *
        * Before delay_count reaches the maximum delay, only Binv is updated with a recursive algorithm
        */
-      template<typename VVT>
-      inline void acceptRow(Matrix<T>& Ainv, int rowchanged, const VVT& psiV)
-      {
-        const T cminusone(-1);
-        const T czero(0);
-        const int norb = Ainv.rows();
-        const int lda_Binv = Binv.cols();
-        std::copy_n(Ainv[rowchanged], norb, V[delay_count]);
-        std::copy_n(psiV.data(), norb, U[delay_count]);
-        delay_list[delay_count] = rowchanged;
-        // the new Binv is [[X Y] [Z x]]
-        BLAS::gemv('T', norb, delay_count+1, cminusone, V.data(), norb, psiV.data(), 1, czero, p.data(), 1);
-        // x
-        T y = -p[delay_count];
-        for(int i=0; i<delay_count; i++)
-          y += Binv[delay_count][i] * p[i];
-        Binv[delay_count][delay_count] = y = T(1) / y;
-        // Y
-        BLAS::gemv('T', delay_count, delay_count, y, Binv.data(), lda_Binv, p.data(), 1, czero, Binv.data()+delay_count, lda_Binv);
-        // X
-        BLAS::ger(delay_count, delay_count, cminusone, Binv[delay_count], 1, Binv.data()+delay_count, lda_Binv, Binv.data(), lda_Binv);
-        // Z
-        for(int i=0; i<delay_count; i++)
-          Binv[delay_count][i] *= -y;
-        delay_count++;
-        // update Ainv when maximal delay is reached
-        if(delay_count==lda_Binv) updateInvMat(Ainv);
-      }
+  template<typename VVT>
+  inline void acceptRow(Matrix<T>& Ainv, int rowchanged, const VVT& psiV)
+  {
+    const T cminusone(-1);
+    const T czero(0);
+    const int norb     = Ainv.rows();
+    const int lda_Binv = Binv.cols();
+    std::copy_n(Ainv[rowchanged], norb, V[delay_count]);
+    std::copy_n(psiV.data(), norb, U[delay_count]);
+    delay_list[delay_count] = rowchanged;
+    // the new Binv is [[X Y] [Z x]]
+    BLAS::gemv('T', norb, delay_count + 1, cminusone, V.data(), norb, psiV.data(), 1, czero, p.data(), 1);
+    // x
+    T y = -p[delay_count];
+    for (int i = 0; i < delay_count; i++)
+      y += Binv[delay_count][i] * p[i];
+    Binv[delay_count][delay_count] = y = T(1) / y;
+    // Y
+    BLAS::gemv('T',
```

Comment:
Is there a way to teach clang format that don't break lines for a function call no matter how long the call is?
If you look for t3_contract in Spline?2?Adoptor.h They are all ugly.
When a function is defined breaking the long list of argument is good, but not the place we call it.

Reply:
Or at least not one argument per line.

Reply:
I would like to keep the reformatting moving. Can we either live with the ugliness or protect these sections?

Reply:
@prckent I'm in favour of merge.
@PDoakORNL do you know if there is such a setting in clang-format?